compiler/circle-eval-diff/src/DataSet.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __CIRCLE_EVAL_DIFF_DATASET_H__
+#define __CIRCLE_EVAL_DIFF_DATASET_H__
+
+#include <string>
+#include <vector>
+
+namespace circle_eval_diff
+{
+
+// Data for each input of a single model
+using Input = std::vector<int8_t>;
+// A set of data for one inference of a single model
+using Data = std::vector<Input>;
```

Comment:
Hmm.. It seems that the file name `Dataset.h` does not match well with the content (Data is a vector, not a set). How about `InputData.h` or `TensorVector.h`?

Or, I think it would be better to remove this file and simply replace `Data` with `std::vector<Tensor>`.